### ISSUE TITLE
Not supported keras/examples with MXNet backend

### ISSUE THREAD

**Issue by roywei** [awslabs/keras-apache-mxnet#79]
Examples need to be working fine or exception properly handled. (Specify the  exact functionality MXNet does not support yet)

- [x] addition_rnn.py
- [x] antirectifier.py
- [x] babi_memnn.py **[Sparse Not supported]**
- [x] babi_rnn.py
- [x] cifar10_cnn.py
- [x] cifar10_cnn_capsule.py
- [x] cifar10_cnn_tfaugment2d.py **[Not supported]**
- [x] cifar10_resnet.py
- [x] conv_filter_visualization.py **[Gradient Not supported]**
- [x] conv_lstm.py **[Conv2DLSTM Not supported]**
- [x] deep_dream.py **[Gradient Not supported]**
- [x] image_ocr.py **[CTC Not supported]**
- [x] imdb_bidirectional_lstm.py
- [x] imdb_cnn.py
- [x] imdb_cnn_lstm.py
- [x] imdb_fasttext.py
- [x] imdb_lstm.py
- [x] lstm_seq2seq.py
- [x] lstm_seq2seq_restore.py
- [x] lstm_stateful.py
- [x] lstm_text_generation.py
- [x] mnist_acgan.py **[Sparse Not supported]**
- [x] mnist_cnn.py
- [x] mnist_dataset_api.py  **[TF Specific Not supported]**
- [x] mnist_denoising_autoencoder.py
- [x] mnist_hierarchical_rnn.py
- [x] mnist_irnn.py
- [x] mnist_mlp.py
- [x] mnist_net2net.py
- [x] mnist_siamese.py
- [x] mnist_sklearn_wrapper.py
- [x] mnist_swwae.py **[Gradient Not supported]**
- [x] mnist_tfrecord.py **[TF Specific Not supported]**
- [x] mnist_transfer_cnn.py
- [x] neural_doodle.py  **[Gradient Not supported]**
- [x] neural_style_transfer.py **[Gradient Not supported]**
- [x] pretrained_word_embeddings.py
- [x] reuters_mlp.py
- [x] reuters_mlp_relu_vs_selu.py 
- [x] variational_autoencoder.py **[Custom Loss Not supported]**
- [x] variational_autoencoder_deconv.py **[Custom Loss Not supported]**

**Comment by sandeep-krishnamurthy** [comment]
@roywei - I was unable to run seq2seq example.

**Comment by roywei** [comment]
lstm_seq2seq.py is working in [latest master](https://github.com/awslabs/keras-apache-mxnet/commit/4857329af3139741d75ead97078b1185be7f896b)

**Comment by kalyc** [comment]
As discussed offline, `babi_memnn.py` & `mnist_acgan.py` uses `sparse_categorical_cross_entropy` which is independent of sparse tensor support. Maybe we should label that more clearly above?

**Comment by roywei** [comment]
@kalyc updated,  we should treat is as a normal operator, but still investigate whether it take sparse tensors just like other normal ops.

**Comment by roywei** [comment]
With #148 , two more examples enabled. (babi_memnn.py and mnist_acgan.py)

**Comment by kalyc** [comment]
With the [PR #153](https://github.com/awslabs/keras-apache-mxnet/pull/153), `reuters_mlp_relu_vs_selu.py` example has been disabled as `AlphaDropout` is not supported by MXNet
